File: app/src/main/java/com/salton123/bookmarksbrowser/XApp.java
package com.salton123.bookmarksbrowser;

import android.app.Application;

import org.xutils.x;

/**
 * User: newSalton@outlook.com
 * Date: 2019/4/24 16:11
 * ModifyTime: 16:11
 * Description:
 */
public class XApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        x.Ext.init(this);
        // Picasso.setSingletonInstance(
        //     new Picasso.Builder(this)
        //         .addRequestHandler(new VideoRequestHandler())
        //         .build()
        // );
    }
}

File: app/src/main/java/com/salton123/eleph/XApp.kt
package com.salton123.eleph

import android.app.Application
import com.bumptech.glide.Glide
import com.salton123.eleph.video.compressor.manager.AdMobManager
import com.salton123.manager.ActivityLifeCycleManager
import org.xutils.x

/**
 * User: newSalton@outlook.com
 * Date: 2019/4/24 16:11
 * ModifyTime: 16:11
 * Description:
 */
class XApp : Application() {
    override fun onCreate() {
        super.onCreate()
        x.Ext.init(this)
        AdMobManager.init()
        ActivityLifeCycleManager.INSTANCE.init(this)
    }


    override fun onLowMemory() {
        super.onLowMemory()
        Glide.get(this).onLowMemory()
    }

    override fun onTrimMemory(level: Int) {
        super.onTrimMemory(level)
        Glide.get(this).onTrimMemory(level)
    }
}